packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -30,8 +30,13 @@ const getDefaultConfig = (isDistributable: boolean): ApmAgentConfig => {
     return {
       active: false,
       globalLabels: {},
+      centralConfig: false,
+      captureHeaders: false,
+      captureBody: false,
+      logUncaughtExceptions: true,
```

Comment:
Note: it's possible to set 
```
elastic.apm.captureBody: 'all'
elastic.apm.captureHeaders: true
```
in config. Shouldn't we forbid this to ensure PII isn't exposed?

Reply:
I wasn't sure how strict we should be here. Though this isn't a supported customer-facing feature yet, we may document it in the future. If so, I thought keeping this as flexible as possible for customers would be best. But we're not building this for that use case right now, so maybe it would be better to just lock it down?

Reply:
Locked this down